announcements/2020-Week-28/aws.md

```diff
@@ -0,0 +1,12 @@
+# :zap: This Week in AWS Releases (July 10, 2020):zap:
+
+[AWS v11.4.1](https://github.com/giantswarm/releases/tree/master/aws/v11.4.1) re-activates the recent AWS [release of high-availability (HA) masters](https://github.com/giantswarm/releases/tree/master/aws/v11.4.0) which enables clusters to have three master nodes in different availability zones instead of one master node. It also includes fix for issues with certain OIDC configurations.
```

Comment:
```suggestion
[AWS v11.4.1](https://github.com/giantswarm/releases/tree/master/aws/v11.4.1) re-activates the recent AWS [release of high-availability (HA) masters](https://github.com/giantswarm/releases/tree/master/aws/v11.4.0) which enables clusters to have three master nodes in different availability zones instead of one master node. It also fixes certain OIDC configuration issues.
```